pywps/inout/inputs.py

```diff
@@ -214,7 +214,11 @@ def from_json(cls, json_input):
         elif json_input.get('href'):
             instance.url = json_input['href']
         elif json_input.get('data'):
-            instance.data = json_input['data']
+            data = json_input['data']
+            # remove cdata tag if it exists (issue #553)
+            if 'CDATA' in data:
+                data = data.replace("<![CDATA[", "").replace("]]>", "")
```

Comment:
I think it would be safer to use a regex here. If the data contains instances of "]]>", this will have unintended consequences.

Reply:
I have added a regex for cdata.